static/js/theme-map/Maps/Map.js

```diff
@@ -552,7 +564,8 @@ class Map {
   }
 
   coordinateIsInVisibleBounds(coordinate) {
-    return this.getVisibleBounds().contains(coordinate);
+    const bounds = this._getVisibleBounds();
+    return bounds ? bounds.contains(coordinate) : false;
   }
 
   setCenterWithPadding(coordinate, animated = false) {
```

Comment:
do we need to guard pixelHight and pixelWidth in this function as well?

Reply:
Isn't it being guarded by the ternary op?

Reply:
which ternary op? I didn't look too deep into the rest of the code, but I see that you guarded pixelHeight/width in setBounds(), I assume setCenterWithPadding() would do something with pixelHeight/width that may cause similar error so the same guard may be needed here